src/azure-cli/azure/cli/command_modules/cdn/_help.py

```diff
@@ -159,6 +159,293 @@
+    short-summary: Name of the rule.
+  - name: --order
+    type: string
+    short-summary: The order of the rule. The order number must start from 0 and consecutive. Rule with higher order will be applied later.
+  - name: --match-variable
+    type: string
+    short-summary: Name of the match condition.
+  - name: --operator
+    type: string
+    short-summary: Operator of the match condition.
+  - name: --selector
+    type: string
+    short-summary: Selector of the match condition.
+  - name: --match-values
+    type: string
+    short-summary: Match values of the match condition (comma separated).
+  - name: --negate-condition
+    type: string
+    short-summary: Describes if this is negate condition or not.
+  - name: --transform
+    type: string
+    short-summary: Describes what transforms are applied before matching. Possible values are Lowercase, Uppercase.
+  - name: --action-name
+    type: string
+    short-summary: Name of the action.
+  - name: --cache-behavior
+    type: string
+    short-summary: Caching behavior for the requests. Possible values are BypassCache, Override, SetIfMissing.
+  - name: --cache-duration
+    type: string
+    short-summary: The duration for which the content needs to be cached. Allowed format is [d.]hh:mm:ss.
+  - name: --header-action
+    type: string
+    short-summary: Action to perform on headers. Possible values are Append, Overwrite, Delete.
+  - name: --header-name
+    type: string
+    short-summary: Name of header.
+  - name: --header-value
+    type: string
+    short-summary: Value of header.
+  - name: --header-action
+    type: string
+    short-summary: Action to perform on headers. Possible values are Append, Overwrite, Delete.
+  - name: --redirect-type
+    type: string
+    short-summary: The redirect type the rule will use when redirecting traffic. Possible values are Moved, Found, TemporaryRedirect and PermanentRedirect.
+  - name: --redirect-protocol
+    type: string
+    short-summary: Protocol to use for the redirect. Possible values are MatchRequest, Http, Https.
+  - name: --custom-hostname
+    type: string
+    short-summary: Host to redirect. Leave empty to use the incoming host as the destination host.
+  - name: --custom-path
+    type: string
+    short-summary: The full path to redirect. Path cannot be empty and must start with /. Leave empty to use the incoming path as destination path.
+  - name: --custom-querystring
+    type: string
+    short-summary: The set of query strings to be placed in the redirect URL. leave empty to preserve the incoming query string.
+  - name: --custom-fragment
+    type: string
+    short-summary: Fragment to add to the redirect URL.
+  - name: --query-string-behavior
+    type: string
+    short-summary: Caching behavior for the requests. Possible values are Include, IncludeAll, Exclude and ExcludeAll.
+  - name: --query-parameters
+    type: string
+    short-summary: query parameters to include or exclude (comma separated).
```

Comment:
we already support --query in azure cli. this parameter is a little confused. can we use --include/--exclude?

Reply:
I'm trying to be consistent with swagger spec and ms portal. I think --include/--exclude could also be confusing.